WordPress/src/main/java/org/wordpress/android/ui/posts/PostPreviewActivity.java

```diff
@@ -258,9 +262,15 @@ private void revertPost() {
     private void publishPost() {
         if (!isFinishing() && NetworkUtils.checkConnection(this)) {
             if (!mPost.isLocalDraft()) {
+                Map<String, Object> properties = new HashMap<>();
+
+                Configuration configuration = WordPress.getContext().getResources().getConfiguration();
+                properties.put("hardware_keyboard", configuration.keyboard != Configuration.KEYBOARD_NOKEYS);
```

Comment:
Since we'll have the `hardware_keyboard` metadata I don't think it's necessary to add properties to these events.